packages/stream_video_flutter/android/src/main/kotlin/io/getstream/video/flutter/stream_video_flutter/service/PictureInPictureHelper.kt

```diff
@@ -41,11 +42,8 @@ class PictureInPictureHelper {
 
                     val params = PictureInPictureParams.Builder()
                     params.setAspectRatio(aspect).apply {
-                        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
-                            setAutoEnterEnabled(true)
-                        }
```

Comment:
Should this never have been here? Or do we have an alternative on the dart side now?

Reply:
Yes, I believe it was never needed. For some reason with this line Pip failed to show on Android 15 (without any error) and without it it works. I also tested older Android and there it also works fine without this line